keps/sig-network/20190125-ingress-api-group.md

```diff
@@ -638,13 +641,24 @@ backend:
   `extensions/v1beta1` until `networking.k8s.io/v1` is available.
 - [x] Update documentation to reference `networking.k8s.io/v1beta1` kubernetes/website#14239
 
-### 1.16
+### 1.18 `networking/v1beta1`
+
+- [ ] Implement [Path as a prefix](#path-as-a-prefix)
+    - [ ] New resource added with default to Implementation Specific
+- [ ] Implement new resource [Alternative backend types](#alternative-backend-types)
+- [ ] Implement [Hostname wildcards](#hostname-wildcards)
+- [ ] Implement new resource for [Ingress class](#ingress-class)
+    - [ ] Defaulting admission controller for [Ingress class](#ingress-class)
+
+### 1.19
 
```

Comment:
Seems to me like “Update documentation to reference `networking.k8s.io/v1`” should be in v1.19. Does that sound right?

I'm basing this on “Announce `networking.k8s.io/v1beta1` Ingress as deprecated”.